### ISSUE TITLE
multiple keyboard

### ISSUE THREAD

**Issue by themesfa** [randyrants/sharpkeys#122]
Hi . How i can apply changes only on my wired keyboard  and do nothing with other keyboard ?
i mean i have multiple keyboard and i want just edit one of theme .

**Comment by themesfa** [comment]
ya i know that but there is a program that doing that by using `hardware id` . but this program only have basic options and i cant define all keys . but worked correctly in my test . i set my wired keyboard hardware id and all options only apply on this keyboard and not effected on my primary keyboard .
i think this program not working with windows registery . 
you can visited this link and test it : 

> http://www.2112design.com/blog/multi-keyboard-remapper/

**Comment by randyrants** [comment]
Right, but SharpKeys is simply a tool that modifies a Registry key that Windows then uses to remap the keys.  If that Registry key has no knowledge of which hardware id is in use, SharpKeys has no way to support it.
